### PR TITLE
Add a recap blog-post for pyho-2024

### PR DESCRIPTION
**Pull request by kafui4k**
## Issue Link 🔗: https://github.com/BlackPythonDevs/blackpythondevs.github.io/issues/554

**Issue**: #554 

## Type of Change

- [ ] Bug fix 🐞
- [ x] New feature/page
- [ ] Documentation update
- [ ] Other

## Description 📋

- **What:** Create a recap post for PyHo 2024

- **Why:** Added 3 images to the images folder in the assets directory.

- **How:** It is a blog post for the Black Python Devs community

## Checklist ✅

- [x] Followed the [Code of Conduct](https://github.com/BlackPythonDevs/blackpythondevs.github.io?tab=coc-ov-file) and [Contribution Guide](https://github.com/BlackPythonDevs/blackpythondevs.github.io/blob/469afd71943738b7209bfe56d5a4e35394fbaafc/CONTRIBUTING.md)
- [ x] Ran `pre-commit run --all`
- [x ] All tests pass locally
- [ ] Added tests (if applicable)
- [ ] Documentation updated (if applicable)

## Additional Notes & Screenshots

Add any additional notes or comments that might be helpful for the reviewers.

### CODE REVIEW

File: _posts/2024-11-02-black-python-devs-pyho-2024.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+featured_image: /assets/images/bpd_pyho_2024.jpg
+---
+
+We’re ecstatically pleased as a community to share in the celebration of the incredible impact of Black Python Devs (BPD) on PyHo 2024 (the regional Python community conference of the people of Volta). Through an unwavering commitment to empower underrepresented/underserved developers.
+
+BPD’s dedication to serving as a safe and inclusive space for Black “Pythonistas” of all skill levels is awe inspiring. Fostering a vibrant and supportive community through providing opportunities for connection, learning, and collaboration.
+
+We appreciate Abigail Afi Gbadago, a BPD executive council member for gracing us with her presence during our #Sponsors talk session. Sharing invaluable insights into BPD’s initiatives and the opportunities for participation and collaboration within our local user group.
+
+![Black Python Devs PyHo 2024](/assets/images/bpd_pyho_2024_2.jpeg)
```
Comment (kjaymiller) [inline _posts/2024-11-02-black-python-devs-pyho-2024.md]: @kafui4k - please use an alt text that is descriptive of the photo such as

"woman on stage looking at Black Python Devs Logo on screen"

File: _posts/2024-11-02-black-python-devs-pyho-2024.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Looking ahead, we envisage to continue our collective journey to shaping the future of Python’s open source ecosystem from a collaboration mindset over competition. BPD’s focus on growth and collaboration aligns perfectly with our shared goals and we look forward to working together to achieve even greater heights for the community both locally and globally.
+
+![Black Python Devs PyHo 2024](/assets/images/bpd_pyho_2024_3.jpg)
```
Comment (kjaymiller) [inline _posts/2024-11-02-black-python-devs-pyho-2024.md]: @kafui4k - please use a descriptive alt text such as "group photo of black python devs attendees"

**Review by kjaymiller — CHANGES_REQUESTED** [review]
There are a couple of photo description changes that need to be made but after that, it's good to go.

File: _posts/2024-11-02-black-python-devs-pyho-2024.md (hunk 2 above)
Comment (kafui4k) [reply _posts/2024-11-02-black-python-devs-pyho-2024.md]: updated

File: _posts/2024-11-02-black-python-devs-pyho-2024.md (hunk 1 above)
Comment (kafui4k) [reply _posts/2024-11-02-black-python-devs-pyho-2024.md]: updated

**Review by kjaymiller — APPROVED** [review]

**Pull request merged**